adapter/stdioLogger/stdioLogger.go

```diff
@@ -72,18 +94,43 @@ func (l *logger) LogAccess(entries []adapter.LogEntry) error {
 }
 
 func (l *logger) log(entries []adapter.LogEntry) error {
-	var errors *me.Error
+	var errors *multierror.Error
+	fields := make([]zapcore.Field, 0, 6)
 	for _, entry := range entries {
-		if err := writeJSON(l.logStream, entry); err != nil {
-			errors = me.Append(errors, err)
+		if entry.LogName != "" {
```

Comment:
Does Zap not handle all of these zero values correctly already? In other words, could we do this without all the checks, like we would for proto3?